kr8s/tests/test_objects.py

```diff
@@ -588,6 +588,21 @@ async def test_all_v1_objects_represented():
         assert get_class(obj["kind"], obj["version"])
 
 
+@pytest.mark.parametrize(
+    "name, object_class",
+    [
+        ("pods", Pod),
+        ("services", Service),
+        ("configmaps", ConfigMap),
+        ("nodes", Node),
```

Comment:
This is what I was seeing fail on `1.26`. When running `kr8s.get("nodes")` it was constructing a object new class instead of using `Node`.